eng/build.ps1

```diff
@@ -14,6 +14,12 @@
 # it's fine to call `build.ps1 -build -testDesktop` followed by repeated calls to
 # `.\build.ps1 -testDesktop`.
 
+if ($PSVersionTable.PSVersion.Major -lt "5") {
+   $version = $PSVersionTable.PSVersion
```

Comment:
Inline this variable. `version $($PSVersionTable.PSVersion) detected`

Reply:
Done